src/test/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActorTest.scala

```diff
@@ -347,80 +314,36 @@ class TaskLauncherActorTest extends AkkaUnitTest with Eventually {
       assert(activeCount(launcherRef) == 0)
     }
 
-    for (
-      update <- TaskConditionMapping.Gone.toSeq.map(reason => TaskStatusUpdateTestHelper.lost(reason, f.runningMarathonInstance))
-        .union(Seq(
-          TaskStatusUpdateTestHelper.finished(f.runningMarathonInstance),
-          TaskStatusUpdateTestHelper.killed(f.runningMarathonInstance),
-          TaskStatusUpdateTestHelper.error(f.runningMarathonInstance)))
-    ) {
-      s"Terminated task (${update.simpleName} with ${update.reason} is removed" in new Fixture {
-        Mockito.when(instanceTracker.instancesBySpecSync).thenReturn(InstanceTracker.InstancesBySpec.forInstances(f.runningMarathonInstance))
```

Comment:
e.g. this task was really testing nothing, because the state in instancetracker is set to running instance so whatever update we send, it always gets the same state...